ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/OptimizeService.java

```diff
@@ -182,17 +190,19 @@ private void clearTableCache(TableIdentifier tableIdentifier) {
     } catch (Throwable t) {
       LOG.debug("failed to delete  " + tableIdentifier + " runtime, ignore", t);
     }
-    try {
-      deleteOptimizeRecord(tableIdentifier);
-      deleteOptimizeTaskHistory(tableIdentifier);
-    } catch (Throwable t) {
-      LOG.debug("failed to delete " + tableIdentifier + " optimize(task) history, ignore", t);
+    if (pureDelete) {
```

Comment:
Should we put `deleteTableOptimizeRuntime(tableIdentifier)` into the `pureDelete` code block? I don't think it's necessary to delete the TableOptimizeRuntime after disabling the table's optimizing.